src/modules/interface/pid.h

```diff
@@ -115,8 +117,8 @@ typedef struct
   float outP;         //< proportional output (debugging)
   float outI;         //< integral output (debugging)
   float outD;         //< derivative output (debugging)
-  float iLimit;       //< integral limit
-  float outputLimit;  //< output limit
+  float iLimit;       //< integral limit, absolute value. '0' means no limit.
+  float outputLimit;  //< total PID output limit, absolute value. '0' means no limit.

```

Comment:
I still don't think it's a great idea to have the outputLimit functionality at all -- if the output of the PID needs to be limited, that suggests the coefficients need to be tuned instead. However, the current position controller makes use of this functionality, and I don't have a way to test out changes to that layer, so I've left it in.

See my comments on issue #185 for some more details.